modules/redis/README.md

```diff
@@ -0,0 +1,136 @@
+# Component: `eks/redis`
+
+This component installs `redis` for EKS clusters. This is a Self Hosted Redis Cluster installed on EKS.
+
+## Usage
+
+**Stack Level**: Regional
+
+Use this in the catalog or use these variables to overwrite the catalog values.
+
+```yaml
+components:
+  terraform:
+    eks/redis:
```

Comment:
Please document the `eks/redis/defaults` here abstract and then an example of how to use it

Reply:
added defaults and derived component example